### PR TITLE
Adapt package publishing for new repo layout

### PR DESCRIPTION
**Pull request by stack72**
This is required due to the new go mod module layout

Also update the version of the Google API as this repo was pulling in
a later version - this is what was breaking the PR

### CODE REVIEW

**Comment by stack72** [comment]
//cc @EvanBoyle

File: scripts/go.mod [hunk 1]
```diff
@@ -2,4 +2,4 @@ module github.com/pulumi/pulumi-gcp/scripts
 
 go 1.13
 
-require github.com/pulumi/pulumi/pkg v0.0.0-20200323215703-1d4a75674b2e
+require github.com/pulumi/pulumi/pkg v0.0.0-20200322194843-61928f04e052
```
Comment (mikhailshilkov) [inline scripts/go.mod]: This seems to be going back? Is that intended? Why so many changes in this PR?

Comment (stack72) [reply scripts/go.mod]: The original PR had picked up an incorrect version of the upstream API causing it to throw a build failure

The issue was resolved by using a replace in the go.mod

This version of pulumi/pkg makes everything the same version in the report now

File: sdk/go/gcp/accesscontextmanager/pulumiManifest.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// *** WARNING: this file was generated by the Pulumi Terraform Bridge (tfgen) Tool. ***
```
Comment (mikhailshilkov) [inline sdk/go/gcp/accesscontextmanager/pulumiManifest.go]: I think you should gitignore these files

Comment (stack72) [reply sdk/go/gcp/accesscontextmanager/pulumiManifest.go]: yeah I am just doing that right now :)

**Comment by EvanBoyle** [comment]
Awesome, thanks for picking this up!

**Pull request merged**